ngraph_bridge/executable.cc

```diff
@@ -135,8 +135,10 @@ Executable::Executable(shared_ptr<Function> func, string device)
     auto& name = m_function->get_friendly_name();
     m_network.serialize(name + ".xml", name + ".bin");
     util::DumpNGGraph(func, name + "_executable");
-    options[InferenceEngine::PluginConfigParams::KEY_DUMP_EXEC_GRAPH_AS_DOT] =
-        name + "_IE_" + m_device;
+    if (m_device == "CPU") {
```

Comment:
Do the other plugins not support this?

Reply:
That's correct. Each plugin has a different way to dump graphs if that.